HOST lewis.lbl.gov
only_i 3
only_code None
force_amber_prep True
dry_run False
1 1z3z,A,2.900,1zod,A,1.800
2 1psi,A,2.920,3ndd,A,1.500
... 3 1psi
/net/chevy/raid1/nigel/amber/hi_lo
/net/chevy/raid1/nigel/amber/hi_lo/ps/1psi
1psi.pdb 4phenix_1psi.pdb
1psi.pdb 4phenix_1psi.pdb
local_archive None
archive PDB None
Reading from local mirror:
  /net/cci/pdb_mirror/pdb/ps/pdb1psi.ent.gz
Model saved to /net/lewis/raid1/scratch1/nigel/amber/hi_lo/ps/1psi/1psi.pdb
Traceback (most recent call last):
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 199, in <module>
    run2(sys.argv[1:])
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/build/../modules/cctbx_project/mmtbx/command_line/fetch_pdb.py", line 155, in run2
    files.append(get_pdb(id, data_type_, mirror, log, format=data_format))
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 213, in get_pdb
    data = fetch(id, data_type, mirror=mirror, format=format, log=log)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/iotbx/pdb/fetch.py", line 150, in fetch
    data = libtbx.utils.urlopen(url)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/modules/cctbx_project/libtbx/utils.py", line 2069, in urlopen
    return urllib2.urlopen(*args, **kwds)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 127, in urlopen
    return _opener.open(url, data, timeout)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 404, in open
    response = self._open(req, data)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 422, in _open
    '_open', req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 382, in _call_chain
    result = func(*args)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1214, in http_open
    return self.do_open(httplib.HTTPConnection, req)
  File "/net/cci-filer2/raid1/xp/phenix/phenix-svn/base/lib/python2.7/urllib2.py", line 1184, in do_open
    raise URLError(err)
urllib2.URLError: <urlopen error [Errno 110] Connection timed out>

==================================================
Summary of pdb4amber for: 1psi.pdb
===================================================
REDUCE returned non-zero exit status: See reduce_info.log for more details

----------Chains
The following (original) chains have been found:
A

---------- Alternate Locations (Original Residues!))

The following residues had alternate locations:
None

---------- Gaps (Renumbered Residues!)
gap of 3.773398 A between HIE 209 and LYS 211

---------- Mising heavy atom(s)

None
not pdb_filename /net/chevy/raid1/nigel/amber/hi_lo/ps/1psi/4phenix_1psi.pdb

	Running AmberPrep : 1psi.pdb


==================================================
Running pdb4amber on 1psi.pdb
==================================================

==================================================
Setting up library files for non-standard residues
==================================================
SCS.mol2 is present. Skipping elbow/antechamber run for this residue.


==================================================
Preparing asu files and 4phenix_1psi.pdb
==================================================

~> tleap -f 1psi_asu_tleap_input_run

Checking output filenames
  file : 1psi_asu.prmtop
Sorry:   Output file is empty : 1psi_asu.prmtop
  Check contents of "leap.log"
